packages/calcite-components/src/utils/dynamicCSSConst.ts

```diff
@@ -0,0 +1,7 @@
+export const getWidth = (width: string, widthScale: string): string => {
```

Comment:
Can you narrow the return type? Applies to both.

Reply:
This just returns a class, so type would be just string, no? @jcfranco